src/tests/dispatch-payload.test.ts

```diff
@@ -195,6 +212,7 @@ Deno.test("DispatchPayload with unhandled events", async (t) => {
       mock.assertSpyCalls(unhandledEventSpy, 0);
       mock.assertSpyCall(defaultSpy, 0, {
         args: [{
+          body: payload.body,
```

Comment:
Another change due to DRYing up the argument extraction from payload: now all handlers get a `body` argument.